docs/products/m3db/howto/write-php.rst

```diff
@@ -0,0 +1,47 @@
+Write Data to M3DB with PHP
+---------------------------
+
+This example writes some data to an M3DB service from PHP, making use of the InfluxDB library.
+
+Variables
+'''''''''
+
+These are the placeholders you will need to replace in the code sample:
+
+=================       =======================================================
+Variable                Description
+=================       =======================================================
+``SERVICE_HOST``        Service hostname, found on the service overview page
+``SERVICE_PORT``        Service port number, found on the service overview page
+``AVNADMIN_PASS``       Password for the ``avnadmin`` user
+=================       =======================================================
+
+Pre-requisites
+''''''''''''''
+
+For this example you will need:
+
+1. The PHP InfluxDB library. You can install this with ``composer``::
+
+    composer require influxdata/influxdb-client-php
+
+
+.. note::
+
+    M3DB supports InfluxDB (v1) protocol so we can use the existing library for this database too.
+
+Code
+''''
+
+Add the following to ``index.php`` and replace the placeholders with values for your project:
```

Comment:
which are the placeholders? Maybe we should specify them (SERVICE_HOST, SERVICE_PORT, AVNADMIN_PASS)

Reply:
They are described on lines 11-17 ... does this answer your question?